### ISSUE TITLE
Arabic Characters Not Rendering Properly in Generated PDFs

### ISSUE THREAD

**Issue by rizwan952** [omaralalwi/Gpdf#11]
Some Arabic text is not properly generated.
But some text is rendered properly.

In HTML
POS TERMINAL
<span>جھاز نقطة البیع</span>

Actual generated text 
![image](https://github.com/user-attachments/assets/9490a2a9-6822-47e1-a4b9-b45780bb8429)

Current Versions
        "php": "^8.2",
        "barryvdh/laravel-dompdf": "^3.0",
        "laravel/framework": "^11.9",
        "omaralalwi/gpdf": "^1.0.4",

**Comment by omaralalwi** [comment]
can you remove barryvdh/laravel-dompdf because it will case conflict with GPDF , then clear config and cache then give me update please .

**Comment by omaralalwi** [comment]
also update your version of GPDF to v1.0.6 , we resolved some issues in new versions

**Comment by rizwan952** [comment]
Yes, I have removed the package and cleared the cache and config, but some Arabic strings are still not proper.
Current packages versions 
    "require": {
        "php": "^8.2",
        "darkaonline/l5-swagger": "^8.6",
        "friendsofphp/consul-php-sdk": "^5.2",
        "laravel/framework": "^11.9",
        "omaralalwi/gpdf": "^1.0.6",
        "simplesoftwareio/simple-qrcode": "^4.2",
        "ext-gd": "*"
    },
![image](https://github.com/user-attachments/assets/d42c2793-bd4b-4066-843c-bd7168fcdccd)

**Comment by omaralalwi** [comment]
please remove this package and it's config and cache , to avoid any conflict with `GPDF` package
@rizwan952
